package.json

```diff
@@ -33,9 +33,11 @@
     "fs-extra": "^1.0.0",
     "http-proxy-agent": "^1.0.0",
     "https-proxy-agent": "^1.0.0",
+    "jsonc-parser": "^0.3.0",
```

Comment:
I don't *think* so. That's actually produced by the VS Code team and published by @aeschli [here](https://www.npmjs.com/package/jsonc-parser).